CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 
+* [CHANGE] Replace `exp/slices` package with `slices` from standard library. #626
```

Comment:
There's no need for a changelog item, as the PR doesn't have any user visible effects, it's pure refactoring.
```suggestion
```